### PR TITLE
RATY-96: Add support for Django >3.2, drop support for Python <3.8

### PR DESCRIPTION
**Pull request by frwickst**
This should fix the current deprecation warnings that the project throws. This should allow the project the become Django 4.0 ready as well.

### CODE REVIEW

**Comment by danipran** [comment]
Hello from the future! I'm taking over this PR.

File: django_ilmoitin/__init__.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+if sys.version_info >= (3, 8):
+    from importlib import metadata
+else:
+    import importlib_metadata as metadata
+
+
+__version__ = metadata.version("django_ilmoitin")
+
+
+if django.VERSION < (3, 2):
```
Comment (charn) [inline django_ilmoitin/__init__.py]: Should we just drop anything below 3.2 since there's no support for that old Django anymore?

Comment (danipran) [reply django_ilmoitin/__init__.py]: Right, now that I read the ticket again, we agreed on dropping support for Django <3.2. Will do that.

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit:

**Pull request merged**